changelog/16.0/16.0.0/summary.md

```diff
@@ -99,6 +101,21 @@ or
 > ALTER TABLE your_table ENGINE=InnoDB;
 ```
 
+### <a id="vttablet-restore-metrics">VTTablet Restore Metrics
+
+As part of the VTTablet Sidecar Schema Maintenance Refactor in v16.0.0, we dropped the `local_metadata` table from the sidecar database schema. This table was storing a couple of metrics related to restores from backup, which have now been lost.
+They have been re-introduced in v17.0.0 as metrics that can be accessed from `/debug/vars`.
```

Comment:
IMO this is not helpful as I don't know what issues this may have caused, what was missing and what the equivalent new metrics are.